contracts/interfaces/morpho/IURD.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+/// @notice This contract allows Metamorpho users to claim their rewards
+interface IURD {
```

Comment:
why not using this [interface](https://github.com/morpho-org/universal-rewards-distributor/blob/main/src/interfaces/IUniversalRewardsDistributor.sol)?

Reply:
Fixed